e2e/testdata/fn-eval/invalid-fn-config-file/pkg/.expected/config.yaml

```diff
@@ -16,3 +16,4 @@ testType: eval
 exitCode: 1
 image: gcr.io/kpt-fn/set-namespace:v0.1
 fnConfig: ../../config.yaml
+stdErr: "wrong Node Kind for  expected: MappingNode was ScalarNode: value: {I am not a valid config file}"
```

Comment:
this is a pretty bad error message, it's not really comprehensible or actionable: go/kpt-errors-style-guide
It's leaking low-level YAML constructs to the end-user.

Can you file an issue for fixing `set-namespace` (it's probably comes from underlying kyaml library though).

Reply:
Having low-level YAML errors is not necessary bad if it's qualified and wordsmitehd as such:

`YAML has an issue: expected a map, but got a scalar value `{I am ...}`

Reply:
I'll file an issue for this and other bad error messages. Thank you for the feedback